src/_data/capability-matrix.yml

```diff
@@ -7,6 +7,8 @@ columns:
     name: Apache Flink
   - class: spark
     name: Apache Spark
+  - class: apex
+    name: Apache Apex
```

Comment:
We have lots of horizontal space available, so right here I think you could just add "(on feature branch)". See my other comments about why.